build.gradle

```diff
@@ -55,8 +40,8 @@ repositories {
 }
 
 dependencies {
-    api platform('com.exactpro.th2:bom:2.10.1')
-    implementation "com.exactpro.th2:grpc-common:3.0.0"
+    api platform('com.exactpro.th2:bom:3.0.0')
+    implementation "com.exactpro.th2:grpc-common:3.1.2"
```

Comment:
Please, make the dependency as api (it will go to compile classpath for those who use this project as dependecy)